test/cluster.test.js

```diff
@@ -47,6 +47,67 @@ describe('#Cluster', function () {
     assert.equal(cluster.resolveTimeout, 30000)
   })
 
+  it('should correctly set all security options', function () {
```

Comment:
Not too sure about this -- if we decide to validate the securityOpts on createInstance this test will have to change

Reply:
After discussion have decided to add the validation. Tests have been adjusted accordingly